pkg/apis/policy/v1/doc.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +k8s:conversion-gen=k8s.io/kubernetes/pkg/apis/policy
+// +k8s:conversion-gen-external-types=k8s.io/api/policy/v1
+// +k8s:defaulter-gen=TypeMeta
+// +k8s:defaulter-gen-input=../../../../vendor/k8s.io/api/policy/v1
+
+// Package policy is for any kind of policy object.  Suitable examples, even if
+// they aren't all here, are policyv1.PodDisruptionBudget, PodSecurityPolicy,
+// NetworkPolicy, etc.
```

Comment:
suggest dropping the examples sentence, since PSP and NetworkPolicy won't end up here